.env.dist

```diff
@@ -24,7 +24,7 @@ DATABASE_URL=sqlite:///%kernel.project_dir%/var/data/bolt.sqlite
 #DATABASE_URL=mysql://db_user:"db_password"@localhost:3306/db_name
 
 # Postgres
-#DATABASE_URL=postgresql://db_user:"db_password"@localhost:5432/db_name?serverVersion=11"
+#DATABASE_URL=postgresql://db_user:"db_password"@localhost:5432/db_name?serverVersion=11&charset=utf8"
```

Comment:
Is `utf8` not the default? Do we need to explicitly set it here?

Reply:
I think it is better to explicitly set here, to force users to think about the charset.
Additionally, it is hardcoded in `config/packages/doctrine.yaml` as `charset: utf8mb4`, which is fine for MySQL but unsupported in PostgreSQL (for as far as I could find). Setting it in the DATABASE_URL ensures an override to UTF8.

Reply:
Ok, fair enough! Does Postgres still handle emoji properly, if it doesn't have `utf8mb4`?

Reply:
Emoji should work as they are UTF-8, I'll have to test it to know for sure. 
From what I understand, MySQL utf-8 is not really full utf-8, it uses 3-bytes per character instead the regular 4byte. I do not know why it was decided, but here lies the origin: [https://github.com/mysql/mysql-server/commit/43a506c0ced0e6ea101d3ab8b4b423ce3fa327d0](url)
That's probably why for MySQL utf8mb4 is required. For postgres. utf8 is full utf8, so 4 byte.   
something to keep in mind in case of problems -> [https://stackoverflow.com/questions/36198847/what-type-should-i-use-to-store-emoji-in-postgresql](url)